tests/test_ai/test_priority_algorithm/test_mutations/test_robinhood.py

```diff
@@ -83,119 +90,191 @@ def test_mutate_robinhood__only_changes_zero_or_two_team_sets(self):
         """
         Either no teams should change or two teams should change.
         """
-
-        priority_team_set, student_dict = create_new_priority_team_set(3, 9)
-        priorities = [TestPriority()]
-
-        mutated_team_set = mutate_robinhood(priority_team_set, priorities, student_dict)
-
-        # Count the number of teams that changed
-        changed_teams = 0
-        for i in range(3):
-            if (
-                get_priority_team(i, priority_team_set).student_ids
-                != get_priority_team(i, mutated_team_set).student_ids
-            ):
-                changed_teams += 1
-
-        self.assertIn(changed_teams, [0, 2], "Exactly 0 or 2 teams should be changed")
+        for mutate_func in [mutate_robinhood, mutate_robinhood_holistic]:
+            priority_team_set, student_dict = create_new_priority_team_set(3, 9)
+            priorities = [StudentListPriority([1, 2])]
+
+            mutated_team_set = mutate_func(priority_team_set, priorities, student_dict)
+
+            # Count the number of teams that changed
+            changed_teams = 0
+            for i in range(3):
+                if (
+                    get_priority_team(i, priority_team_set).student_ids
```

Comment:
That's ok though because if the order changed, that means that the mutate function decided to mutate this team at some point, which counts as a changed team for this test